configs/webserver/sample.conf

```diff
@@ -98,6 +98,8 @@ max_file_upload_size = 4294967296
 # e.g. cr.backend.ai/stable/python
 # You should pick default_environment in ui section too.
 #allowlist = ""
+# Show all environment images regardless of installation
+show_non_installed_images = false
```

Comment:
According to the internal rule to **use negations sparingly**, I suggest to rename this options.
e.g. `show_installed_images_only = true`